src/user/ZulipDevAuthView.js

```diff
@@ -72,12 +69,14 @@ class ZulipDevAuthView extends React.Component {
             />
           )}
         </ScrollView>
+        <ZulipError errors={this.props.errors} />
       </View>
     );
   }
 }
 
 const mapStateToProps = (state) => ({
+  auth: state.auth,
   account: state.user.accounts.get(state.user.activeAccountId),
```

Comment:
Looks like `activeAccountId` has been removed but there's still a reference to it here